fact_extractor/plugins/unpacking/jffs2/test/test_plugin_jffs2.py

```diff
@@ -11,11 +11,9 @@ def test_unpacker_selection_generic(self):
         self.check_unpacker_selection('filesystem/jffs2-big', 'JFFS2')
 
     def test_extraction_little(self):
-        self.check_unpacking_of_standard_unpack_set(
-            TEST_DATA_DIR / 'jffs2_be.img', additional_prefix_folder='jffs-root'
-        )
+        meta_data = self.check_unpacking_of_standard_unpack_set(TEST_DATA_DIR / 'jffs2_be.img')
```

Comment:
Was it by design, that the test regarding the seemingly big endian file, was called _little while the seemingly little endian test is called _big?

Reply:
Yes that seems weird. According to git, you wrote that part, though :sweat_smile: 
I changed it